src/runtimes/node/index.ts

```diff
@@ -108,6 +109,10 @@ const zipFunction: ZipFunction = async function ({
     bundlerWarnings,
     config,
     inputs,
+    includedFiles: await getFilteredPathsOfIncludedFiles(
```

Comment:
Am I right in thinking that each bundler is already calling `getPathsOfIncludedFiles` and we're now adding a second call, at the runtime level? Would it make sense to have each bundler return this information to the runtime?

Reply:
The problem is, that for the esbuild and zisi bundler `getPathsOfIncludedFiles` is called inside `getSrcFiles`. In order to return included_files from the bundler without an additional call, I would need to change the return type of `getSrcFiles`. So instead of an array of paths return `{srcFiles:[], included_paths:[]}`. Do you think it is worth changing this?

Reply:
Ah, I see! I'm okay with shipping as is for now and maybe create an issue to revisit later (or not revisit at all, if we don't think it's worth doing). Happy to defer to you.

Reply:
It was less work than anticipated. I thought the impact of changing this would have been bigger. So now `getSrcFiles` returns an object.